aten/src/ATen/native/Embedding.cpp

```diff
@@ -35,8 +35,9 @@ namespace at { namespace native {
 
 Tensor embedding_symint(const Tensor & weight, const Tensor & indices,
                         c10::SymInt padding_idx, bool scale_grad_by_freq, bool sparse) {
-  TORCH_CHECK(weight.dim() == 2,  "'weight' must be 2-D");
-  auto indices_arg = TensorArg(indices, "indices", 1);
```

Comment:
Not sure why `weight` doesn't use helper functions like the rest of the file. So I've updated it.

Also, the convention in the file is for arguments to be 1-indexed. I've updated `1` to `2` for `indices`. But now I wonder whether it's this way because `nn.functional.embedding` has `input` and `weight` swapped.